bug-finding.ipynb

```diff
@@ -234,6 +346,38 @@
     "# Bug 2 analysis and response"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": 97,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "26932"
+      ]
+     },
+     "execution_count": 97,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "lemmata = [t.lemma_ for t in df['nlp_docs'].explode()] \n",
+    "counter = Counter(lemmata)\n",
+    "observed_freq_the = counter['the']\n",
+    "\n",
+    "observed_freq_the\n",
+    "## 26,932 occurences of 'the'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The original code didn't work because .find() is not a count function. I don't understand the approach original code was taking, so my \"fix\" may not be what you are looking for. But instead of tweaking it, I tried a different method to get the frequency using the Counter function to count the occurences of the lemma 'the.' "
```

Comment:
A fix is a fix! Your approach works great, and yes, `find()` is definitely not the right function to have used in the first case.